eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1484,6 +1484,46 @@ public function testCopySubtree()
     }
 
     /**
+     * Test for the copySubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::copySubtree()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testCopySubtreeWithAliases()
+    {
+        $repository = $this->getRepository();
+        $urlAliasService = $repository->getURLAliasService();
+
+        // $mediaLocationId is the ID of the "Media" page location in
+        // an eZ Publish demo installation
+
+        // $demoDesignLocationId is the ID of the "Demo Design" page location in an eZ
+        // Publish demo installation
+        $mediaLocationId = $this->generateId('location', 43);
+        $demoDesignLocationId = $this->generateId('location', 56);
+
+        $locationService = $repository->getLocationService();
+        $locationToCopy = $locationService->loadLocation($mediaLocationId);
+        $newParentLocation = $locationService->loadLocation($demoDesignLocationId);
+
+        $expectedSubItemAliases = [
+            '/Design/Plain-site/Media/Multimedia',
+            '/Design/Plain-site/Media/Images',
+            '/Design/Plain-site/Media/Files',
+        ];
+
+        $this->assertAliasesBeforeCopy($urlAliasService, $expectedSubItemAliases);
+
+        // Copy location "Media" to "Design"
```

Comment:
A bit late, and not omportant, but if you had named `$locationToCopy` and `$newParentLocation` after what they are instead, the comment would not have been necessary.

`$locationService->copySubtree($mediaLocation, $demoDesignLocation)`